roles/docker/tasks/package_docker.yml

```diff
@@ -93,7 +93,7 @@
     dest: /etc/sysconfig/docker
     regexp: '^OPTIONS=.*$'
     line: "OPTIONS='\
-      {% if ansible_selinux.status | default(None) == '''enabled''' and docker_selinux_enabled | default(true) %} --selinux-enabled {% endif %}\
+      {% if ansible_selinux.status | default(None) == 'enabled' and docker_selinux_enabled | default(true) | bool %} --selinux-enabled {% endif %}\
```

Comment:
Let me see if I understand this: without `bool`, setting `docker_selinux_enabled=False` in the inventory would make the variable a string with value `"False"`, which is a truthy value in Python leading to always true no matter what string/value you set in the inventory. Is that right?